edgar_tool/cli.py

```diff
@@ -19,7 +18,8 @@ def _validate_text_search_args(
     min_wait_secs: float,
     max_wait_secs: float,
     retries: int,
-    browser_name: str,
+    browser_name: Optional[str],
```

Comment:
any reason to deprecate it slowly versus outright eliminating it from the source code? Bearing in mind it means more maintenance down the road where you actually drop it.

Reply:
This is a good point.

I mostly don't want another major version change at this point, it feels like there might be other changes to make before a V2 makes sense. Think I'm going to keep it as is, but will deal with this in the future.